doc/user_guide/docker/docker_usage.md

```diff
@@ -0,0 +1,135 @@
+# Using Exasol AI-Lab Docker Edition
+
+Using Exasol AI-Lab Docker Edition requires some specific prerequisites but also offers additional benefits.
+
+## Need to Know About Docker Images and Containers
+
+Exasol AI-Lab Docker Edition is published as a so-called _Docker Image_ on [Docker Hub](https://hub.docker.com/r/exasol/data-science-sandbox).
+
+In order to use such an image you need two components
+* Docker client
+* Docker service, aka. _Docker daemon_
+
+The client must run on your local machine, the daemon can be on the same machine, a remote machine, or inside a virtual machine.
```

Comment:
the must part for the client is reality a bit strong, but I am not sure if we should change it